f8a_version_comparator/base.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+# Copyright © 2018 Red Hat Inc.
+#
+# This program is free software: you can redistribute it and/or modify
```

Comment:
This is a license header of GPLv3+, but `setup.py` declares license as `ASL 2.0`. So which one is correct? :)